<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkPathListButtonsWidget</class>
 <widget class="QWidget" name="ctkPathListButtonsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Path list</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="AddFilesButton">
     <property name="toolTip">
      <string>Add files</string>
     </property>
     <property name="text">
      <string>Add files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="AddDirectoryButton">
     <property name="toolTip">
      <string>Add a directory</string>
     </property>
     <property name="text">
      <string>Add directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="RemoveButton">
     <property name="toolTip">
      <string>Remove selected entries</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="EditButton">
     <property name="toolTip">
      <string>Edit current entry</string>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
